README.md

```diff
@@ -9,26 +9,70 @@
+
+![](./docs/imgs/kyuubi_positioning.png)
+
+- [x] A HiveServer2-like API
+- [x] Multi-tenant Spark Support
+- [x] Running Spark in a serverless way
+
+
+### Target Users
+
+Kyuubi's goal is to make it easy and efficient for `anyone` to use Spark(maybe other engines soon) and facilitate users to handle big data like ordinary data. Here, `anyone` means that users do not need to have a Spark technical background but a human language, SQL only. Sometimes, SQL skills are unnecessary when integrating Kyuubi with Apache Superset, which supports rich visualizations and dashboards.
+
+
+In typical big data production environments with Kyuubi, there should be system administrators and end-users.
+
+- System administrators: A small group consists of Spark experts responsible for Kyuubi deployment, configuration, and tuning.
+- End-users: Focus on business data of their own, not where it stores, how it computes.
+
+Additionally, the Kyuubi community will continuously optimize the whole system with various features, such as History-Based Optimizer, Auto-tuning, Materialized View, SQL Dialects, Functions, e.t.c.
+
+
+### Usage scenarios
+
+#### Port workloads from HiveServer2 to Spark SQL
+
+In typical big data production environments, especially secured ones, all bundled services manage access control lists to restricting access to authorized users. For example, Hadoop YARN divides compute resources into queues. With Queue ACLs, it can identify and control which users/groups can take actions on particular queues. Similarly, HDFS ACLs control access of HDFS files by providing a way to set different permissions for specific users/groups.
+
+Apache Spark is a unified analytics engine for large-scale data processing. It provides a Distributed SQL Engine, a.k.a, the Spark Thrift Server(STS), designed to be seamlessly compatible with HiveServer2 and get even better performance.
+
+HiveServer2 can identify and authenticate a caller, and then if the caller also has permissions for the YARN queue and HDFS files, it succeeds. Otherwise, it fails. However, on the one hand, STS is a single Spark application, and the user and the queue to which STS belongs are uniquely determined at startup. Consequently, STS cannot leverage cluster managers such as YARN and Kubernetes for resource isolation and sharing or control the access for callers by the single user inside the whole system. On the other hand, the Thrift Server is coupled in the Spark driver's JVM process. This coupled architect puts a high risk on server stability and makes it unable to handle high client concurrency or apply high availability such as load balancing as it is stateful.
```

Comment:
>>> However, on the one hand, STS is a single Spark application, and the user and the queue to which STS belongs are uniquely determined at startup.
?
However, on the one hand, STS is a single Spark application, the user and the queue, to which STS belongs, are uniquely determined at startup.

Reply:
However, on the one hand, STS is a single Spark application. The user and queue to which STS belongs are uniquely determined at startup.

How about？

Reply:
LGTM